Dockerfile

```diff
@@ -0,0 +1,15 @@
+# edge is requried to get the fix for https://bugs.alpinelinux.org/issues/4451
+FROM alpine:edge
+MAINTAINER Tom Denham <tom@projectcalico.org>
+
+# The final container is slightly bloated by having the source code
+# But it's only a couple of MB (kept down using .dockerignore)
+ADD . /code
+WORKDIR /code
+
+RUN apk -U add python py-setuptools libffi ip6tables ipset iputils && \
+    apk add --virtual temp python-dev libffi-dev py-pip alpine-sdk && \
+    pip install -e . && \
+    apk del temp && rm -rf /var/cache/apk/*
+RUN mkdir /etc/calico && echo -e "[global]\nMetadataAddr = None\nLogFilePath = None\nLogSeverityFile = None" >/etc/calico/felix.cfg
```

Comment:
I agree but I wasn't sure where to put it so that it doesn't confuse people. Any suggestions?